build.cake

```diff
@@ -16,8 +16,8 @@ var ErrorDetail = new List<string>();
 // SET PACKAGE VERSION
 //////////////////////////////////////////////////////////////////////
 
-var version = "3.3.0";
-var modifier = "";
+var version = "3.4.0";
+var modifier = "-alpha-1";
```

Comment:
Do you plan to do alphas as well as betas for this one? Seems like we are more beta-like since we won't be adding any more features to 3.4.